libs/@local/eslint-config/generate-ignore-patterns.cjs

```diff
@@ -8,28 +8,28 @@ const monorepoRoot = path.resolve(__dirname, "../../..");
  * Because .*ignore files are not composable, we cannot import or otherwise reuse
  * a top-level .eslintignore. To avoid repetition and to maintain a coherent behavior
  * of ESLint CLI and IDE extensions, we generate ignore patterns for each workspace
- * based from .gitignore. This is done via ignorePatterns option in ESLint config.
+ * based from .prettierignore. This is done via ignorePatterns option in ESLint config.
```

Comment:
Switching ignore list source because of a new section called "Shared between linters". We can ignore `.yarn/` in linters, but we still need to track `.yarn/releases/*` by git.